app/views/design-system/rsd-design-system/design-patterns/trust-details/index.html

```diff
@@ -0,0 +1,149 @@
+        
+        <p>Whichever design you use, the trust details section should:</p>
+        <ul>
+          <li>have more than one trust details field</li>
+          <li>provide a change link in the third column, if the fields are editable</li>
+          <li>point to one edit page, even if you have a change link for each field</li>
+        </ul>
+        
+        <p>It should <strong>not</strong>:</p>
+        <ul>
+          <li>allow users to edit fields on the same page. For example, using AJAX-enabled forms. Take users to a separate edit page</li>
+          <li>provide a change link if users are not allowed to edit the information. This may be the case if a certain user group only needs to read it</li>
+        </ul>
+        
+        <h2>Editing trust details</h2>
+        
+        <p>If users are allowed to edit the trust details, create an editing page and allow users to access it via the change link(s).</p>
+        
+        <h3>Retrieving trust details</h3>
+        
+        <p>Some users want to search for a trust using a trust reference number (TRN), trust ID, or trust name.</p>
+        
+        <p>If this is the case, you can use a text input box and primary action button.</p>
+
+        <img src="public/assets/images/trn-trust-reference-number.png" alt class="A text input for TRN (trust reference number) with a primary action button.">
+        
+        <p>When users search for the trust, provide a page to allow users to confirm the retrieved information matches what they need.</p>
+
+         <img src="public/assets/images/is-this-the-correct-trust.png" alt class="A table with trust details. Underneath the table is a radio button question, Is this the correct trust?.">
+        
+        <p>When the user selects Yes, direct them to the relevant page in their user journey. If they select No, direct them back to the trust search page.</p>
+        
+        <h3>Entering trust details manually</h3>
+        
+        <p>There could be trust information that needs to be entered manually, rather than retrieved from a search. For example, the name and email address of a contact in the trust.</p>
+        
+        <p>In this case, add extra rows in your summary list or card, and let users enter the information manually. See the <a href="https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_wiki/wikis/Academies-and-Free-Schools-SIP.wiki/7376/Contact-details">contact details design pattern</a>.</p>
+        
+        <h2>Research on this pattern</h2>
+        
+        <p>We have not done research on this design pattern, but the pattern is being used to meet these needs from our <a href="https://educationgovuk.sharepoint.com/sites/ServiceDeliveryDirectorate/Lists/RSD%20crossprogramme%20user%20needs/AllItems.aspx?env=WebViewList">user needs database</a>:</p>
```

Comment:
Intranet link

Reply:
Thanks for flagging it. We have just removed the entire section for now but we will definitely pay attention to that problem in other parts of the website.